clusters.py

```diff
@@ -320,6 +336,23 @@ def MakeDataClass(column_names: List,
 
   return data
 
+def CheckBCandAC(data: pd.DataFrame, threshold: int = 100) -> pd.DataFrame:
+  """
+  A function to drop the rows where Bone Conduction is greater than Air 
+  Conduction by atleast 10dB
```

Comment:
Comment is wrong, it's no longer 10dB.  You should also explain that the default, 100, means that no data is removed.